components/crud-web-apps/jupyter/frontend/cypress/e2e/form-page.cy.ts

```diff
@@ -57,4 +57,61 @@ describe('New notebook form', () => {
           });
       });
   });
+
+  it('should update panel header name according to the name input field', () => {
+    cy.visit('/new');
+
+    cy.get('[data-cy="workspace volume"]').click();
+
+    cy.get('[data-cy="workspace volume"]')
+      .find('[data-cy="volume name input"]')
+      .clear()
+      .type('test');
+
+    cy.get('[data-cy="workspace volume"]')
+      .find('[data-cy="volume name header"]')
+      .contains('test');
+  });
+
+  it('should update name input field according to the ConfigMap', () => {
+    cy.visit('/new');
+    cy.wait(['@mockConfigRequest']);
+
+    cy.get('[data-cy="workspace volume"]').click();
+
+    cy.get('[data-cy="workspace volume"]')
+      .find('[data-cy="volume name input"]')
+      .should($nameInput => {
+        const nameValue = $nameInput.val();
+        //
+        expect(nameValue).equal('-workspace'); // '{notebook-name}-workspace' is the name value of the config fixture
```

Comment:
The intention here was probably to move the comment from line 87 to where the `//` are now? Nit but I think it 'd be better this way in order to avoid having a really long line.